### ISSUE TITLE
Feature Request: Open Page in New Tab

### ISSUE THREAD

**Issue by mmarder96** [alyssaxuu/omni#122]
## Feature Request: Open Page in New Tab

I often want to open a bookmark or other page in a new tab from the Omni command palette but there currently is no option to do that. I would love it if we could open the currently selected option in a new tab using `Shift+Enter` or something.

**Comment by chase-miller** [comment]
+1 for this. It would be awesome to make the behavior configurable. I'm basically trying to emulate Arc's command bar in Firefox, and I've set cmd + t to launch omni.

**Comment by buttercannfly** [comment]
You can now use command/ctrl + enter to open the page in new tab, more advanced features can be found in https://github.com/buttercannfly/AIpex

**Comment by buttercannfly** [comment]
> +1 for this. It would be awesome to make the behavior configurable. I'm basically trying to emulate Arc's command bar in Firefox, and I've set cmd + t to launch omni.

Yeah command + t is the best shorcut for omni. However, chrome doesn't allow extension to override it, now I build a new verison in https://github.com/buttercannfly/AIpex , you can modify the shortcut after installed, hope you like it
![image](https://github.com/user-attachments/assets/695fcabf-e8cc-4bba-83b3-cf2ec518162e)
